### ISSUE TITLE
The Markdown editor field does not have associated labels

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1208]
The input field for the Markdown editor component does not have an associated label. Use label for= or aria-label so that screenreaders have appropriate context.

**Comment by rmmayo** [comment]
addressed as part of #1175
